cucumber-jparest/src/main/java/uk/gov/homeoffice/digital/sas/cucumberjparest/stepdefinitions/ApiSteps.java

```diff
@@ -0,0 +1,163 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparest.stepdefinitions;
+
+import io.cucumber.java.en.When;
+import java.util.Map;
+import lombok.NonNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.HttpResponseManager;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.JpaRestApiClient;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.JpaRestApiResourceResponse;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.JpaRestApiResponse;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.PayloadManager;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.PayloadManager.PayloadKey;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.Persona;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.Resource;
+
+/**
+ * Provides steps related to making API requests to endpoints exposed by the JpaRest package.
+ */
+public class ApiSteps {
+
+  private final HttpResponseManager httpResponseManager;
+  private final JpaRestApiClient jpaRestApiClient;
+  private final PayloadManager payloadManager;
+
+  @Autowired
```

Comment:
This class should be defined as a bean e.g `@Component` since we're autowiring in dependencies.

Reply:
Adding @Component annotation causes the following error:
[ERROR]   Glue class uk.gov.homeoffice.digital.sas.cucumberjparest.stepdefinitions.PayloadSteps was annotated with @Component; marking it as a candidate for auto-detection by Spring. Glue classes are detected and registered by Cucumber. Auto-detection of glue classes by spring may lead to duplicate bean definitions. Please remove the @Component annotation

I've removed @Autowired annotation